src/mito2/src/config.rs

```diff
@@ -68,12 +70,18 @@ pub struct MitoConfig {
     // Other configs:
     /// Buffer size for SST writing.
     pub sst_write_buffer_size: ReadableSize,
+    /// Parallelism to scan a region (default: 1/4 of cpu cores).
+    ///
+    /// 0 or 1 to disable parallelism scan.
+    pub scan_parallelism: usize,
+    /// Capacity of the channel to send data from parallel scan tasks to the main task (default 32).
+    pub parallel_scan_channel_size: usize,
```

Comment:
Can we just make `scan_parallelism` and `parallel_scan_channel_size` as a linear multiplicative relationship, that is to say, `parallel_scan_channel_size` = n * `scan_parallelism'?

Reply:
Each parallel scan task creates a channel so they won't share the same channel. Typically the default channel size is enough.